recipes/rjsmin/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rjsmin" %}
+{% set version = "1.0.12" %}
+{% set sha256 = "dd9591aa73500b08b7db24367f8d32c6470021f39d5ab4e50c7c02e4401386f1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
It looks like this is building a C-extension, so please add `toolchain` to the `build` section.